manifests/g/Geert-JanBesjes/wxMUN/0.40/Geert-JanBesjes.wxMUN.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Geert-JanBesjes.wxMUN
+PackageVersion: 0.40
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://nav.dl.sourceforge.net/project/wxmun/wxmun/0.40/wxmun-0.40-setup-win32.exe

```

Comment:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/wxmun/files/wxmun/0.40/wxmun-0.40-setup-win32.exe/download
```

Reply:
why? isnt .exe ok?

Reply:
Direct link to the `.exe` file will always be better than the mirror link because the direct link automatically chooses the mirror based on the server where you are so you can download it faster.

Reply:
I am new to winget, do I need to resubmit my pacage or can I just update this one, if so how can I do it? Thanks for your help!

Reply:
Just click the "Commit suggestion" button next to my review.